src/components/LangSelector/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface ILangSelector {
+  langueages: string[];
```

Comment:
i think you could replace string[], with more specific type which describe every language, for example 
enum Language {
  us: 'us'.
  fr: 'fr'
}